tests/parser/test_relion_pipeline.py

```diff
@@ -138,12 +139,13 @@ def test_relion_pipeline_current_jobs_property_with_timing_info_multiple_jobs(
     ]
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
 def test_get_pipeline_jobs_from_preprocess_log_file(dials_data):
     pipeline = RelionPipeline("Import/job001")
     pipeline.load_nodes_from_star(
         dials_data("relion_tutorial_data") / "default_pipeline.star"
     )
     preproc_jobs = pipeline._get_pipeline_jobs(
-        dials_data("relion_tutorial_data") / "pipeline_PREPROCESS.log"
+        pathlib.Path(dials_data("relion_tutorial_data") / "pipeline_PREPROCESS.log")
```

Comment:
```suggestion
        dials_data("relion_tutorial_data", pathlib=True) / "pipeline_PREPROCESS.log"
```
[There is a new feature in dials-data](https://dials-data.readthedocs.io/en/latest/installation.html#as-a-developer-to-write-tests-with-dials-data), and this could be the first place to use it :grin:
(At some point it'll become the default)

Reply:
Ah, missed that. Useful, thanks